UnitySDK/Assets/ML-Agents/Examples/3DBall/Prefabs/Game.prefab

```diff
@@ -315,10 +329,9 @@ MonoBehaviour:
   brain: {fileID: 11400000, guid: 383c589e8bb76464eadc2525b5b0f2c1, type: 2}
   agentParameters:
     agentCameras: []
+    agentRenderTextures: []
```

Comment:
This should be fine, right? I guess the file predates adding the field to the AgentInfo.